tests/macros/utils.js

```diff
@@ -154,11 +178,31 @@ function afterEachMacro(teardown) {
     });
 }
 
+/**
+ * Reads a JSON fixture file from the `fixtures` directory.
+ *
+ * @param {...string} filePath
+ */
+function readJSONFixture(...filePath) {
```

Comment:
Do you think that fixtures will always be JSON-formatted and that tests will always want them in parsed form? What if you wanted to use a fixture file for a string that you were going to pass to mockTemplate()?

I think it would be more versatile to rename this to readFixture() and have it return a string. Then individual tests can decide whether to call parseJSON() or not. Or I suppose you could just define both readFixture() and readJSONFixture()

Reply:
I’ve added `readFixture` which doesn’t automatically add the file extension.